docs/docs/user/references/defaults.md

```diff
@@ -2,16 +2,20 @@
 
 KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults). This file provides the user with the power to set defaults for any and all [components](/user/references/components), thus omitting the need to repeat the same settings in [`pipeline.yaml`](/user/references/components).
 
+A very important mechanic of KPOps' defaults is that defaults set for a component apply to all components that inherit from it. An example can be found [here](https://github.com/bakdata/kpops-examples/tree/main/word-count/deployment/kpops/defaults).
+
 ??? "KPOps component Hierarchy"
     --8<--
     ./docs/resources/architecture/components-hierarchy.md
     --8<--
 
-A very important mechanic of KPOps' defaults is that defaults set for a component apply to all components that inherit from it. An example can be found [here](#examples).
+!!! tip inline end
+    `defaults` is the default value of `defaults_filename_prefix`.
+    It, together with `defaults_path` and  `environment` can be changed in [`config.yaml`](/user/references/config)
```

Comment:
```suggestion
    Together with `defaults_path` and `environment` it can be changed in [`config.yaml`](/user/references/config).
```
reads a little easier :)